src/spyglass/decoding/visualization_2D_view.py

```diff
@@ -317,14 +302,16 @@ def create_2D_decode_view(
     if head_dir is not None:
         head_dir = np.squeeze(np.asarray(head_dir))
 
-    track_width, track_height, upper_left_points = make_track(
-        position, bin_size=bin_size
-    )
+    track_bin_width = track_bin_dimensions[0]
+    track_bin_height = track_bin_dimensions[1]
+    # NOTE: We expect caller to have converted from fortran ordering already
+    # i.e. somewhere upstream, centers = env.place_bin_centers_[env.is_track_interior_.ravel(oder="F")]
```

Comment:
```suggest
# i.e. somewhere upstream, centers = env.place_bin_centers_[env.is_track_interior_.ravel(order="F")]
```

Reply:
Sorry, this was just fixing the spelling of order in the comment (as per the linting)

Reply:
Ah, gotcha. Fixed